docs/source/common_issues.rst

```diff
@@ -808,3 +808,78 @@ not necessary:
     class NarrowerArgument(A):
         def test(self, t: List[int]) -> Sequence[str]:  # type: ignore[override]
             ...
+
+TypeVars are not the same
+------------------------------------------------
+
+As discussed in the issue
+`TypeVars should be allowed to be the same <https://github.com/python/mypy/issues/7864>`_,
+TypeVars are *not* equivalent even if they are identically defined.
+
+Here is an example using two TypeVars that are unconstrained and unbound:
```

Comment:
Nit: I'd leave out "unconstrained and unbound" since most type variables are like this in my experience, and not all users are familiar with constraints and bounds, possibly causing some confusion.